app/src/main/java/com/checkin/app/checkin/session/scheduled/ScheduledSessionCartView.kt

```diff
@@ -315,6 +350,18 @@ class ScheduledSessionCartView @JvmOverloads constructor(
     override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
     }
 
+    private fun setupTaxDetails(taxDetailModel: TaxDetailModel) {
+        balloonTaxDetails.getContentView().apply {
+            tv_tax_cgst.text = "₹ ${taxDetailModel.cgst}"
+            tv_tax_sgst.text = "₹ ${taxDetailModel.sgst}"
+            if (taxDetailModel.igst.toFloat() != 0F) {
+                tv_tax_igst.visibility = View.VISIBLE
+                tv_tax_igst_heading.visibility = View.VISIBLE
+                tv_tax_igst.text = "₹ ${taxDetailModel.igst}"
+            }
+        }
+    }
+
```

Comment:
Hey @shivanshs9, Is this okay or should I use `findViewById()` ?

Reply:
:+1: